aws-logs-metricfilter/aws-logs-metricfilter.json

```diff
@@ -38,6 +61,22 @@
           "pattern": ".{1,100}",
           "minLength": 1,
           "maxLength": 100
+        },
+        "MetricUnit": {
```

Comment:
This should just be Unit, to match the api: https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutMetricFilter.html

Reply:
updated